packages/frontend/component/src/ui/input/input.tsx

```diff
@@ -82,11 +68,12 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
     >
       {preFix}
       <input
+        autoFocus={autoFocus}
         className={clsx(input, {
           large: size === 'large',
           'extra-large': size === 'extraLarge',
         })}
-        ref={autoFocus ? handleAutoFocus : upstreamRef}
```

Comment:
Can you check other places where this component is used is work fine?

Reply:
It seems something was wrong after I modified.This part of the code has been reset.